Update Settings Pages (#2242)
hashicorp/terraform-website: content/cloud-docs/users-teams-organizations/organizations.mdx

# Organizations

Organizations are a shared space for one or more [teams][] to collaborate on workspaces.

## API and Terraform Enterprise Provider

In addition to the Terraform Cloud UI, you can use the following methods to manage organizations:
- [Organizations API](/cloud-docs/api-docs/organizations)
- The `tfe` provider [`tfe_organization`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/organization) resource

## Selecting Organizations

Terraform Cloud displays your current organization in the left of the top navigation bar. To select an organization:

1. Click the current organization name to view a list of all the organizations where you are a member.
1. Click an organization to select it. Terraform Cloud displays list of workspaces within that organization.

## Joining and Leaving Organizations

> To join an organization, you must be invited by one of its [owners][] and must accept the emailed invitation. Refer to [Organization Settings: Users](#users) for details.

Reviewer: ```suggestion
To join an organization, you must be invited by one of its owners and must accept the emailed invitation. Refer to [Organization Settings: Users](#users) for details.
```
Not sure if those brackets were meant to be included or not.

Author: same as below! they are supposed to be here, but this is an unusual pattern, so I appreciate you calling it out!

[permissions-citation]: #intentionally-unused---keep-for-maintainers

You can leave an organization from your user account settings. Refer to [User Settings: Organizations](/cloud-docs/users-teams-organizations/users#organizations) for details.

## Creating Organizations

On Terraform Enterprise, administrators can restrict your ability to create organizations. Refer to [Administration: General Settings](/enterprise/admin/application/general#organization-creation) for details.

> On Terraform Cloud, any user can create a new organization. If you do not belong to any organizations, Terraform Cloud prompts you to create one the first time you log in. Otherwise, to create an organization:

Reviewer: ```suggestion
On Terraform Cloud, any user can create a new organization. If you do not belong to any organizations, Terraform Cloud prompts you to create one the first time you log in. To create an organization:
```
It might just be me, but I can't see the two clauses that 'otherwise' is linking here.

Author: yeah this is fair. I think I was just trying to be like "you will do this automatically if you don't already have an org" but I feel like it was poorly done. Good edit!

1. Click the current organization name and select **Create new organization**. The **Create a new organization** page appears.
1. Enter a unique **Organization name** Organization names can include numbers, letters, underscores (`_`), and hyphens (`-`).
1. Provide an **Email address** where you will receive notifictions about the organization.
1. Click **Create organization**.

Terraform Cloud shows the new organization in the navigation bar and prompts you to create a new workspace. You can also [invite other users](#users) to join the organization.

## Managing Settings

To view and manage an organization's settings, click **Settings** in the top navigation bar.

The contents of the organization settings depends on your permissions within the organization. All users can review the organization's contact email, view the membership of any teams they belong to, and view the organization's authentication policy. [Organization owners][owners] can view and manage the entire list of organization settings. Refer to [Organization Permissions](/cloud-docs/users-teams-organizations/permissions#organization-permissions) for details.

> You may be able to manage the following types of organization permissions.

Reviewer: ```suggestion
You may be able to manage the following organization permissions.
```

[permissions-citation]: #intentionally-unused---keep-for-maintainers

### Organization Settings

#### General

> Review the organization name and contact email. Organization owners can also  change the organization name and contact email or delete the organization.

Reviewer: ```suggestion
Review the organization name and contact email. Organization owners can also change the organization name and contact email or delete the organization.
```

!> **Warning:** Deleting or renaming an organization can be very disruptive. We strongly recommend against deleting or renaming organizations with active members.

To rename an organization that manages infrastructure:

> 1.  Alert all members of the organization about its impending deletion.

Reviewer: ```suggestion
1. Alert all members of the organization about its impending deletion.
```

Author: yeah it is!! thank you

1. Cancel in progress and pending runs or wait for them to finish. Terraform Cloud cannot change the name of an organization with runs in progress.
1. Lock all workspaces to ensure that no new runs will start before you change the name.
1. Rename the organization.
1. Update all components using the Terraform Cloud API to the new organization name. This includes Terraform's `cloud` block CLI integration, the `tfe` Terraform provider, and any external API integrations.
1. Unlock workspaces and resume normal operations.

#### Plan & Billing

Review the organization's plan and any invoices for previous plan payments. Organization owners can also upgrade to one of Terraform Cloud's paid plans, downgrade to a free plan, or begin a free trial of paid features.

#### Tags

Review a list of tags for all resources across the organization. When you delete a tag from this page, Terraform Cloud removes it from all resources.

#### Teams

> -> **Note:** Team management is a paid feature, available as part of the **Team** upgrade plan. Free organizations only include an owners team that can include up to five members. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.

Reviewer: ```suggestion
-> **Note:** Team management is a paid feature, available as part of the **Team** upgrade plan. Free plan organizations only include an owners team that can include up to five members. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
```

> All users in an organization can access the **Teams** page, which displays a list of [teams][] within the organiation. This excludes [secret teams](/cloud-docs/users-teams-organizations/teams#team-visibility) where you are not a member. You can also view team membership and manage team API tokens.

Reviewer: ```suggestion
All users in an organization can access the **Teams** page, which displays a list of teams within the organiation. This excludes [secret teams](/cloud-docs/users-teams-organizations/teams#team-visibility) where you are not a member. You can also view team membership and manage team API tokens.
```

I wasn't sure if those brackets should be included or not.

Author: ahh that's a fair question! This is a weird linking pattern that is kind of legacy throughout the docs. So in this case yes, it should be. They define what this link is at the top of the page and then we can reuse it throughout without having to re-enter the link path each time (it's pretty convenient actually).

> Organization owners can also create and delete teams, manage team membership, and manage team API tokens. Remember that users must accept an invitation to the organization before you can add them to teams.

Reviewer: I really like that we're reminding them here, because they do often forget!

[permissions-citation]: #intentionally-unused---keep-for-maintainers

#### Users

Organization owners can invite Terraform Cloud users into the organization, cancel invitations, and remove existing members.

> The list of users has one tab for active users and one tab for invited users who have not yet accepted their invitations. For active users, the list includes usernames, email addresses, avatar icons, two-factor authentication status, and current team memberships. Use the **Search by username or email** field to filter these lists.

Reviewer: ```suggestion
The list of users is separated into one tab for active users and one tab for invited users who have not yet accepted their invitations. For active users, the list includes usernames, email addresses, avatar icons, two-factor authentication status, and current team memberships. Use the **Search by username or email** field to filter these lists.
```
Is the list of invited-but-not-accepted users worth mentioning in terms of contents?

Variable sets let you reuse the same variables across multiple workspaces in the organization. For example, you could define a variable set of provider credentials and automatically apply it to several workspaces, rather than manually defining credential variables in each. Changes to variable sets instantly apply to all appropriate workspaces, saving time and reducing errors from manual updates.

Refer to the [variables overview](/cloud-docs/workspaces/variables) documentation for details about variable types, scope, and precedence. Refer to [managing variables](/cloud-docs/workspaces/variables/managing-variables) for details about how to create and manage variable sets.

### Integrations

#### Cost Estimation

-> **Note:** Cost estimation is a paid feature, available as part of the **Team & Governance** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.

Enable and disable the [cost estimation](/cloud-docs/cost-estimation) feature for all workspaces.

#### Policies

> This is a deprecated interface for managing Sentinel policies. Use the policy sets page instead.

Reviewer: ```suggestion

```
Is this even worth keeping if it is deprecated?

Author: That's a fair point, Dane. It's still in the UI, so I think we do need to document it so folks aren't confused. When they take it out of the UI, I can take it out of the docs!

#### Policy Sets

-> **Note:** Sentinel policies are a paid feature, available as part of the **Team & Governance** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.

Create groups of [Sentinel policies](/cloud-docs/sentinel) from a connected VCS repository and assign those policy sets to workspaces. Sentinel is an embedded policy-as-code framework that can enforce rules about Terraform runs within an organization. You must have [permission to manage policies](/cloud-docs/users-teams-organizations/permissions).

[permissions-citation]: #intentionally-unused---keep-for-maintainers

#### Run Tasks

Manage the run tasks that you can add to workspaces within the organization. [Run tasks](/cloud-docs/workspaces/settings/run-tasks) let you integrate third-party tools and services at specific stages in the Terraform Cloud run lifecycle.

### Security

#### Agents

-> Note: Terraform Cloud Agents are a paid feature, available as part of the **Business** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.

> Create and manage [Terraform Cloud agent pools](/cloud-docs/agents). Terraform Cloud Agents let Terraform Cloud to communicate with isolated, private, or on-premises infrastructure.  This is useful for on-premises infrastructure types such as vSphere, Nutanix, OpenStack, enterprise networking providers, and infrastructure within a protected enclave.

Reviewer: ```suggestion
Create and manage [Terraform Cloud agent pools](/cloud-docs/agents). Terraform Cloud Agents let Terraform Cloud communicate with isolated, private, or on-premises infrastructure. This is useful for on-premises infrastructure types such as vSphere, Nutanix, OpenStack, enterprise networking providers, and infrastructure within a protected enclave.
```

#### SSH Keys

Manage [SSH keys for cloning Git-based modules](/cloud-docs/workspaces/settings/ssh-keys) during Terraform runs. This does not include keys to access a connected VCS provider.

#### SSO

-> **Note:** Single sign-on is a paid feature, available as part of the **Business** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.

Organization owners can [set up an SSO provider](/cloud-docs/users-teams-organizations/single-sign-on) for the organization.

### Version Control

#### General

Workspaces that use part of a shared repository do not typically run plans for changes that do not affect their files. This includes [speculative plans](/cloud-docs/run/remote-operations#speculative-plans) on pull requests. Since **pending** status checks can block pull requests, workspaces automatically send passing commit statuses for any PRs that do not affect their files.

You can disable this behavior if it creates too many status checks to your VCS provider. You may want to do this if you have a large number of workspaces sharing one VCS repository.

#### VCS Events

> -> **Note:** This feature is in beta.

Reviewer: ```suggestion
-> **Note:** This feature is in beta as of May 2022.
```
On features like this, is it worth mentioning a timestamp of some kind to bookmark when the docs were made, or is the feature's release status being tracked in some other channel?

Author: this is an interesting question. We don't typically put times into the docs like this if we can help it. Theoretically, the docs should be up-to-date. BUT thank you for flagging this because I actually do want to flag whether this is still in beta.....there are some inconsistencies in the UI
